genet/core.py

```diff
@@ -1839,6 +1839,20 @@ def has_node(self, node_id):
     def has_nodes(self, node_id: list):
         return all([self.has_node(node_id) for node_id in node_id])
 
+    def has_isolated_nodes(self) -> bool:
+        return bool(self.isolated_nodes())
```

Comment:
Not sure if this is needed as a separate function?

Reply:
it's just a convenience method, makes the code more readable later on